code/game/atoms.dm

```diff
@@ -145,6 +145,9 @@
 		stack_trace("Warning: [src]([type]) initialized multiple times!")
 	flags_1 |= INITIALIZED_1
 
+	if(loc)
+		SEND_SIGNAL(loc, COMSIG_ATOM_ENTERED, src) // if the new item is created over a turf, send a signal that it entered that turf, `loc` in this case

```

Comment:
I believe I said before you need to use a new signal, this is not the entered proc

Reply:
Oh do mean I should make a new signal name, `COMSIG_ATOM_CREATED` or something like that?

Reply:
Sure